src/cache/mod.rs

```diff
@@ -387,39 +362,40 @@ impl Cache {
         self.guilds.iter().map(|i| *i.key()).chain(unavailable_guild_ids).collect()
     }
 
-    /// Retrieves a [`Channel`] from the cache based on the given Id.
-    ///
-    /// This will search the `channels` map, then the [`Self::private_channels`] map.
-    ///
-    /// If you know what type of channel you're looking for, you should instead manually retrieve
-    /// from one of the respective methods:
+    /// Retrieves a [`GuildChannel`] from the cache based on the given Id.
     ///
-    /// - [`GuildChannel`]: [`Self::guild_channel`] or [`Self::guild_channels`]
-    /// - [`PrivateChannel`]: [`Self::private_channel`] or [`Self::private_channels`]
+    /// This will search the guild cache for the [`Guild::channels`] map, then the
+    /// `Self::temp_channels` map if enabled and not found.
```

Comment:
This should be reworded since `Cache::temp_channels` isn't a method and the field is only `pub(crate)`. Plus, it's feature-gated.